dune-project

```diff
@@ -476,6 +476,8 @@ For more information see https://semgrep.dev
     uuidm
     (timedesc (>= 2.0.0)) ; used via git-wrapper
     (lsp (= 1.15.1-5.0))
+    git
+    git-unix
```

Comment:
Will this be alright for Windows support?

Reply:
🤷 there's a mirage version that I *think* has js_of_ocaml support. We can do some virtual modules stuff if this gets in the way

Reply:
ok

Reply:
One of our CI job is checking whether semgrep is building on windows.

Reply:
build-test-windows-x86.yml, so if the workflow pass that means we're fine using git and git-unix